runnvda.bat

```diff
@@ -0,0 +1,2 @@
+@echo off
+call "%~dp0\venvUtils\venvCmd.bat" py "%~dp0\source\nvda.pyw" %*
```

Comment:
If called from a batch file, pythonw will actually block. Calls in batch files always wait on any process executed. And since it blocks, and I added the freeConsole call in core.py so that the console is still usable by NVDA, we may as well use python rather than pythonw so that critical errors are not missed.
However, I have now also added runnvdaw.bat which executes NvDA in the virtual environment with start pyw, which again provides the behaviour developers were used to by using pyw or executing nvda.pyw directly from Explorer etc. I.e. it does not block, and no console window stays around for the life time of its execution.
Of course this being a draft pr, all this is still up for debate -- I certainly welcome other ideas on the best way we can allow developers to run NVDA.